### ISSUE TITLE
Loculated fetal ascites

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7217]
**Preferred term label:**
Loculated fetal ascites

**Synonyms**

**Definition (free text, please give PubMed ID)**

Comment: Fluid can become loculated when confined by adhesions, malignancy or infection. (https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5438051/)
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
Loculated ascites may become encapsulated and form a collection. This can potentially exert a localized mass effect.

**Comment by SophieCollardeau** [comment]
peritoneal fluid collection, septated ascites: synonym?
loculated fetal ascites  can also be observed in inborn error of metabolism such as lysosomal storage disease
meconium peritonities can sometimes be cystic called meconium pseudocyst
doi: 10.1016/j.ajog.2003.09.016. PMID: 14981382: article with causes of non-immune fetal ascites

**Comment by pnrobinson** [comment]
Suggestion
No need for separate fetal term, instead this:

Def: A type of ascites (peritoneal fluid collection)  that is divided into loculi (compartments) by septa (fibrous walls). Ascites fluid can become loculated when confined by adhesions, malignancy or infection.
Synonym: Septated ascites
Parent: Ascites

PMID:28567101
PMID:14981382

**Comment by pnrobinson** [comment]
Label: Meconium pseudocyst
Def: A meconium-containing cyst in the peritoneal space . Meconium pseudocysts form in the presence of meconium peritonitis, which  is defined as a sterile peritonitis caused by escape of meconium from the intestinal tract into the general peritoneal cavity during the fetal or perinatal period. When the extruded meconium becomes walled off, it can form a rim-calcified mass representing the meconium pseudocyst.
PMID: 19293041
Comment: A fetal intraperitoneal cystic mass that shows high T1 and low T2 signals in prenatal magnetic resonance imaging is considered pathognomonic of meconium pseudocyst. See Figure 1 of PMID: 19293041.
Parent - Abnormal fetal morphology for now, consider more hierarchy

**Comment by pnrobinson** [comment]
added terms
